example.py

```diff
@@ -50,78 +50,61 @@ def Image(props):
     })
 
 
-class Link:
-    def __init__(self, use_new_window=False):
-        self.use_new_window = use_new_window
-
-    def render(self, props):
-        link_props = {
-            'href': props['url'],
-        }
-
-        if self.use_new_window:
-            link_props['target'] = '_blank'
-            link_props['rel'] = 'noreferrer noopener'
-
-        return DOM.create_element('a', link_props, props['children'])
+def link(props):
+    return DOM.create_element('a', {
+        'href': props['url']
+    }, props['children'])
```

Comment:
Just a note that we're losing the `use_new_windows` functionality here, although it was never used since no arguments were ever passed to the elements in `DOM.create_element`.

Reply:
I'll quickly address how to achieve similar results in the changelog.